jfr-container-action/action.yml

```
@@ -40,7 +40,7 @@ runs:
     - name: Move cache
       if: steps.cache-jenkins.outputs.cache-hit == 'true'
       run: ls -al /jenkins_new_plugins && cp -r /jenkins_new_plugins /usr/share/jenkins/ref/plugins
-      shell: bash
+      shell: bash   
```

Suggest getting rid of the white spaces introduced in the PR

```suggestion
      shell: bash
```